### PR TITLE
Fix RWC Runner to report both .types and .symbols errors

### PR DESCRIPTION
**Pull request by yuit**

### CODE REVIEW

File: src/harness/harness.ts [hunk 1]
```diff
@@ -1385,8 +1385,15 @@ namespace Harness {
                 e2 = e;
             }
 
-            if (e1 || e2) {
-                throw e1 || e2;
+            if (e1 && e2) {
```
Comment (DanielRosenwasser) [inline src/harness/harness.ts]: instead of `e1` and `e2`, make this `typeErrors` and `symbolErrors`

File: src/harness/harness.ts [hunk 2]
```diff
@@ -1385,8 +1385,15 @@ namespace Harness {
...
-            if (e1 || e2) {
-                throw e1 || e2;
+            if (e1 && e2) {
+                throw new Error(e1.message + ts.sys.newLine + e2.message);
+            }
+
+            if (e1) {
+                throw e1;
+            }
+            else if(e2) {
```
Comment (DanielRosenwasser) [inline src/harness/harness.ts]: Just make this `if`, and add a space after the `if`

File: src/harness/harness.ts [hunk 3]
```diff
@@ -1396,7 +1403,12 @@ namespace Harness {
 
                 const fullExtension = isSymbolBaseLine ? ".symbols" : ".types";
 
-                Harness.Baseline.runBaseline(baselinePath.replace(/\.tsx?/, fullExtension), () => fullBaseLine, opts);
+                // When calling this function from rwc-runner, it is possible that the baselinePath will have no extension.
```
Comment (DanielRosenwasser) [inline src/harness/harness.ts]: When is this possible? Leave this in the comment.

Comment (DanielRosenwasser) [inline src/harness/harness.ts]: Unless the following two are such cases - the wording doesn't make this clear.

**Comment by yuit** [comment]
:bell:

**Pull request merged**